node/src/rust/web/events_info.rs

```diff
@@ -0,0 +1,247 @@
+use axum::{
+    extract::{
+        ws::{Message, WebSocket, WebSocketUpgrade},
+        State,
+    },
+    response::Response,
+    routing::get,
+    Router,
+};
+use futures::StreamExt;
+use serde_json::{json, Value};
+
+use shared::rust::shared::{f1r3fly_event::F1r3flyEvent, f1r3fly_events::EventStream};
+use tracing::error;
+
+use crate::rust::web::shared_handlers::AppState;
+
+pub struct EventsInfo;
+
+impl EventsInfo {
+    pub fn create_router() -> Router<AppState> {
+        Router::new().route("/", get(events_info_handler))
+    }
+
+    async fn handle_websocket(mut socket: WebSocket, mut event_stream: EventStream) {
```

Comment:
Scala send 'startedEvent':
```scala
    val startedEvent = Text(
      Json
        .obj(
          ("event", Json.fromString("started")),
          (schemaVersionLabel, schemaVersionJson)
        )
        .noSpaces
    )
```

Just double checking, is that sending anywhere? if not, can we add something like that?

```suggestion
    async fn handle_websocket(mut socket: WebSocket, mut event_stream: EventStream) {
      // Send initial "started" event
      let started = json!({
          "event": "started",
          "schema-version": 1
      });
      if let Ok(msg) = serde_json::to_string(&started) {
          let _ = socket.send(Message::Text(msg.into())).await;
      }
```